### PR TITLE
quickstart: add ECS quickstart

### LINKED ISSUE #56: add quickstart for ECS
write a simple document that gets someone up and running with Bottlerocket on ECS

### PR DESCRIPTION
**Pull request by mchaker**
**Issue number:**

Closes #56 

**Description of changes:**

Adds a quickstart for ECS. Builds on top of the EKS quickstart PR (please merge that first).

**Terms of contribution:**

By submitting this pull request, I confirm that my contribution is made under
the terms of the licenses outlined in the LICENSE-SUMMARY file.

### CODE REVIEW

File: content/en/os/1.13.x/install/quickstart/aws/ecs/index.markdown [hunk 1]
```diff
@@ -0,0 +1,126 @@
++++
+title="Amazon ECS"
+type="docs"
+description="How to get started with Bottlerocket on Amazon ECS"
++++
+
+## Introduction
+
+In this quickstart, we use a number of techniques and tools like `jq` and environment variables to make the experience as simple and straightforward as possible.
```
Comment (stockholmux) [inline content/en/os/1.13.x/install/quickstart/aws/ecs/index.markdown:9]: too many 'ands': 'techniques and tools' -> 'techniques/tools' 
more specific: 'the experience' -> ' the quickstart experience'

File: content/en/os/1.13.x/install/quickstart/aws/ecs/index.markdown [hunk 2]
```diff
@@ -0,0 +1,126 @@
++++
+title="Amazon ECS"
+type="docs"
+description="How to get started with Bottlerocket on Amazon ECS"
++++
+
+## Introduction
```
Comment (stockholmux) [inline content/en/os/1.13.x/install/quickstart/aws/ecs/index.markdown:7]: I don't think we really need this heading: remove it.

File: content/en/os/1.13.x/install/quickstart/aws/ecs/index.markdown [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+## Prerequisites
+
+- Complete the steps in [Setting up with Amazon ECS](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/get-set-up-for-amazon-ecs.html).
+- Ensure that the AWS user you use for this quickstart has the permissions specified in the [Amazon ECS First Run Wizard Permissions](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/security_iam_id-based-policy-examples.html#first-run-permissions) IAM policy example.
+- Ensure that you have an [IAM role created](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/instance_IAM_role.html#instance-iam-role-create) named `ecsInstanceRole` and configured according to the AWS ECS Developer Guide.
+  - Be sure to follow _both_ the [AWS Console steps](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/instance_IAM_role.html#instance-iam-role-create) as well as the AWS CLI steps (section labeled "To create the ecsInstanceRole role (AWS CLI)" - the AWS CLI steps cover an important aspect: adding an Instance Profile to the role)
+- [`aws-cli`](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html#getting-started-install-instructions): Used to interact with AWS services.
```
Comment (stockholmux) [inline content/en/os/1.13.x/install/quickstart/aws/ecs/index.markdown:18]: I think we might want to split this up. The first four bullets are activities, the last two are software. Add a sentence that describes the activities, then go into the bullets. Then add a sentence that describes the required software and go into the list.

File: content/en/os/1.13.x/install/quickstart/aws/ecs/index.markdown [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+export AWS_REGION=us-west-2
+export ECS_CLUSTER_NAME=bottlerocket-quickstart-ecs
+export ECS_INSTANCE_ROLE_NAME=ecsInstanceRole
+```
+
+Then, create an ECS cluster with the AWS CLI as follows:
+
+```bash
+aws ecs --region $AWS_REGION create-cluster --cluster-name $ECS_CLUSTER_NAME
+```
```
Comment (stockholmux) [inline content/en/os/1.13.x/install/quickstart/aws/ecs/index.markdown:35]: what's the expected results for this?

File: content/en/os/1.13.x/install/quickstart/aws/ecs/index.markdown [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+## Launching Instances Into Your Cluster
+
+After your ECS cluster is created, you can launch instances into it.
```
Comment (stockholmux) [inline content/en/os/1.13.x/install/quickstart/aws/ecs/index.markdown:39]: The proposition 'into' with launch is making me 🧐

Would it be better to _add_ instances instead?

File: content/en/os/1.13.x/install/quickstart/aws/ecs/index.markdown [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+
+```bash
+aws iam attach-role-policy \
+   --role-name $ECS_INSTANCE_ROLE_NAME \
+   --policy-arn arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
+```
+
+### Connecting To Your Cluster
+
+In order for the nodes that you launch to be able to communicate with ECS, you need to make sure to configure the nodes with the name of the ECS cluster.
```
Comment (stockholmux) [inline content/en/os/1.13.x/install/quickstart/aws/ecs/index.markdown:53]: too many words. 

'In order for the nodes that you launch to be able to communicate with ECS, you need to make sure to configure the nodes with the name of the ECS cluster.' -> 'In order ~~for the nodes that you launch to be able~~ to communicate with ECS, ~~you need to make sure to~~ configure ~~the nodes~~ __each node__ with the name of the ~~ECS~~ cluster.'

File: content/en/os/1.13.x/install/quickstart/aws/ecs/index.markdown [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+cat << EOF > quickstart-ecs-user-data.toml
+[settings.ecs]
+cluster = "$ECS_CLUSTER_NAME"
+EOF
+```
+
+### Launch
+
+Now you can launch Bottlerocket instances into your ECS cluster!
+There are a few values that you need to set in your environment first:
```
Comment (stockholmux) [inline content/en/os/1.13.x/install/quickstart/aws/ecs/index.markdown:66]: This is complex enough to need explanation. Tell the reader that you're automatically grabbing the information with aws cli.

**Review by stockholmux — CHANGES_REQUESTED** [review]
A few language notes and areas that need fleshing out.

File: content/en/os/1.13.x/install/quickstart/aws/ecs/index.markdown (hunk 5 above)
Comment (mchaker) [reply content/en/os/1.13.x/install/quickstart/aws/ecs/index.markdown:39]: Yes! Changing that.

**Review by stockholmux — APPROVED** [review]
LGTM

**Pull request merged**